### PR TITLE
Licenses.txt to attributions.yml

### LINKED ISSUE #15607: license.txt files need be converted to RGA/yml
Each of these files just need to be converted into an `attributions.yml` file. For an example file, see [ Resources/Audio/Items/attributions.yml](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Items/attributions.yml). Instead of opening one giant PR, this issue should be fixed by several smaller PRs (e.g., one per category/folder).

- [x] [ Resources/Audio/Misc/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Misc/license.txt)
- [x] [ Resources/Audio/Items/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Items/license.txt)
- [x] [ Resources/Audio/Animals/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Animals/license.txt)
- [x] [ Resources/Audio/Ambience/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Ambience/license.txt)
- [x] [ Resources/Audio/Machines/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Machines/license.txt)
- [x] [ Resources/Audio/Voice/Talk/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Voice/Talk/license.txt)
- [x] [ Resources/Audio/Voice/Human/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Voice/Human/license.txt)
- [x] [ Resources/Audio/Weapons/Xeno/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Weapons/Xeno/license.txt)
- [x] [ Resources/Audio/Announcements/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Announcements/license.txt)
- [x] [ Resources/Audio/Items/Medical/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Items/Medical/license.txt)
- [x] [ Resources/Audio/Voice/Skeleton/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Voice/Skeleton/license.txt)
- [x] [ Resources/Audio/Effects/Vehicle/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Effects/Vehicle/license.txt)
- [x] [ Resources/Audio/Items/Handcuffs/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Items/Handcuffs/license.txt)
- [x] [ Resources/Audio/Voice/Reptilian/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Voice/Reptilian/license.txt)
- [x] [ Resources/Audio/Ambience/Objects/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Ambience/Objects/license.txt)
- [x] [ Resources/Audio/Effects/Diseases/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Effects/Diseases/license.txt)
- [x] [ Resources/Audio/Ambience/Temporary/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Ambience/Temporary/license.txt)
- [x] [ Resources/Audio/Weapons/Guns/Gunshots/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Weapons/Guns/Gunshots/license.txt)
- [x] [ Resources/Audio/Announcements/RoundEnd/license.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Announcements/RoundEnd/license.txt)
- [x] [ Resources/Audio/Magic/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Magic/licenses.txt)
- [x] [ Resources/Audio/Effects/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Effects/licenses.txt)
- [x] [ Resources/Audio/Weapons/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Weapons/licenses.txt)
- [x] [ Resources/Audio/Effects/Cargo/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Effects/Cargo/licenses.txt)
- [x] [ Resources/Audio/Effects/Shuttle/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Effects/Shuttle/licenses.txt)
- [x] [ Resources/Audio/Effects/Weather/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Effects/Weather/licenses.txt)
- [x] [ Resources/Audio/Effects/Lightning/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Effects/Lightning/licenses.txt)
- [x] [ Resources/Audio/Weapons/Guns/Hits/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Weapons/Guns/Hits/licenses.txt)
- [x] [ Resources/Audio/Weapons/Guns/Misc/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Weapons/Guns/Misc/licenses.txt)
- [x] [ Resources/Audio/Weapons/Guns/Miss/licenses.txt](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Weapons/Guns/Miss/licenses.txt)
- [x] Bonus issue: fix `emergency_meeting.ogg` licence (or just remove the effect altogether)

### PR DESCRIPTION
**Pull request by HighTechPuddle**
## About the PR
I started converting licenses.txt to attributions.yml. Woo was that a mistake, well doesn't matter, so there are some issues with it:

1. Some files aren't mentioned at all anywhere so there is no way to credit them.
2. Some files are wrongly licensed and some don't even have a proper source.
3. We will have to find replacements for some files because we cant even use them, bellow will be a list of sounds we cant use or are missing a license or I can't find the author/source.

Files may need replacing:

> at /Audio/Items/Toys
    weh.ogg, muffled_weh.ogg, skub.ogg: License: CC0-1.0, dont know the author, figured the skub is SS14 only so the source is this repo,

> at /Audio/Items/Medical
    healthscanner.ogg: license: CC-BY-4.0, By FM Synthesis, cant find source, just so the validator would accept it I just put github,com there,

> at /Audio/Animals
lizard_happy.ogg: video is deleted, need a replacement.
bear.ogg: video is deleted, need a replacement.
penguin_squawk.ogg: no license stated in the video, might need a replacement.

And some questions:

- Can I make a entry in the wiki on how to properly add stuff to attributions files?

Closes/Fixes #15607
Closes/Fixes #3413

### CODE REVIEW

**Comment by mirrorcult** [comment]
`pop.ogg` was created and added by me, the soniss sounds should also all be fine: https://sonniss.com/gameaudiogdc

**Comment by HighTechPuddle** [comment]
> `pop.ogg` was created and added by me, the soniss sounds should also all be fine: https://sonniss.com/gameaudiogdc

Nice, so what license should I use for your sound or any other of your sounds? The one most assets use: CC-BY-SA 3.0? Or MIT which is the license for this whole repo?

**Comment by mirrorcult** [comment]
CC0 1.0

**Comment by HighTechPuddle** [comment]
Alr so we still have a problem with that last one. Should I contact the creator and ask him/her if we can use this one? Or should I stay quite until we find a replacement?

**Comment by mirrorcult** [comment]
would rather just replace sounds esp if they're super generic

**Comment by HighTechPuddle** [comment]
Found a good enough replacement under the CC0 1.0 here [freesound.org](https://freesound.org/people/florianreichelt/sounds/460509/), will cut it and add it to this DPR.

**Comment by HighTechPuddle** [comment]
I am aware of the many errors, working on it rn.

**Comment by HighTechPuddle** [comment]
FINALLY MY GOD.

**Comment by HighTechPuddle** [comment]
Alr so the only issues left should be the ones listed above, also that one font we cant use it. Like really we cant its paid.

**Comment by HighTechPuddle** [comment]
No we can I am just stupid.

**Comment by HighTechPuddle** [comment]
Oh yeah it can be merged now

**Comment by HighTechPuddle** [comment]
Everything should be alr, if it isnt just create an issue (after this gets merged) and mention me.

**Comment by HighTechPuddle** [comment]
Alr fuck me I guess, WHY IS IT NOT WORKING?!? I ONLY ADDED .YML FILES AND ONE SOUND FILE WHY IS NOT COMPILING PROPERLY?!?!?!

**Comment by HighTechPuddle** [comment]
My God. I will look into it when I get some sleep and comeback from school. Look through every single file again to make sure everything is how it should be.

File: Resources/Audio/Animals/attributions.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+- files: ["goat_bah.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://youtu.be/QIhwzsk5bww"
+
+- files: ["lizard_happy.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://youtu.be/I7CX0AS8RNI"
```
Comment (ficcialfaint) [inline Resources/Audio/Animals/attributions.yml:104]: This video is unavailable

File: Resources/Audio/Animals/attributions.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+- files: ["bear.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://www.youtube.com/watch?v=pz6eZbESlU8"
```
Comment (ficcialfaint) [inline Resources/Audio/Animals/attributions.yml:109]: This video is unavailable

File: Resources/Audio/Animals/attributions.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+- files: ["raccoon_chatter.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://youtu.be/BGjFP1CP7E0"
+
+- files: ["goat_bah.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://youtu.be/QIhwzsk5bww"
```
Comment (ficcialfaint) [inline Resources/Audio/Animals/attributions.yml:99]: It doesn’t say its license anywhere

File: Resources/Audio/Animals/attributions.yml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+- files: ["space_dragon_roar.ogg"]
+  license: "CC-BY-SA-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://github.com/tgstation/tgstation"
+
+- files: ["penguin_squawk.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Taken or edited from source"
+  source: "https://youtu.be/Anr35RbBL3Y"
```
Comment (ficcialfaint) [inline Resources/Audio/Animals/attributions.yml:89]: No license stated in the video

**Comment by metalgearsloth** [comment]
You should really break this pr up coz it's going to be conflict bait and take time to address reviews.

File: Resources/Audio/Animals/attributions.yml (hunk 4 above)
Comment (HighTechPuddle) [reply Resources/Audio/Animals/attributions.yml:89]: License was stated in the license.txt but I will try to find the original and confirm the license.

**Comment by HighTechPuddle** [comment]
> You should really break this pr up coz it's going to be conflict bait and take time to address reviews.

How can I do that? The conflicts are easy to resolve anyway.

File: Resources/Audio/Animals/attributions.yml (hunk 3 above)
Comment (HighTechPuddle) [reply Resources/Audio/Animals/attributions.yml:99]: Will do the same.

Comment (HighTechPuddle) [reply Resources/Audio/Animals/attributions.yml:99]: Can't find a way to contact the author will find I replacement.

File: Resources/Audio/Animals/attributions.yml (hunk 4 above)
Comment (HighTechPuddle) [reply Resources/Audio/Animals/attributions.yml:89]: Cant find any good replacement. I will add it to the list

File: Resources/Audio/Animals/attributions.yml (hunk 3 above)
Comment (HighTechPuddle) [reply Resources/Audio/Animals/attributions.yml:99]: done.

File: Resources/Audio/Animals/attributions.yml (hunk 2 above)
Comment (HighTechPuddle) [reply Resources/Audio/Animals/attributions.yml:109]: Couldn't find a replacement adding it to the list.

File: Resources/Audio/Animals/attributions.yml (hunk 1 above)
Comment (HighTechPuddle) [reply Resources/Audio/Animals/attributions.yml:104]: Added to the list.

**Review by ElectroJr — CHANGES_REQUESTED** [review]
Thanks for helping clean up the licenses.

> How can I do that? The conflicts are easy to resolve anyway.

You can, and should, just open multiple smaller PRs. Instead of doing all of these audio files in one giant PR, these should've been broken up, e.g., one PR for `/Ambience/*`, one for `/Items/*`, etc. Creating a smaller PR makes it easier both for us to review and means you'll get feedback earlier on and avoid potential doing unnecessary work that needs to be changed later on. It also means you won't be hit by a sudden barrage of issues & feedback.

On that note, many of the licences changes need to be reworked. E.g., you converted 

```
﻿shipambience.ogg from /tg/station (commit https://github.com/tgstation/tgstation/blob/66a625e6df15eaa97e599248a2281c74238ce26e/sound/ambience/shipambience.ogg), which took it from CEV Eris (commit https://github.com/discordia-space/CEV-Eris/blob/e4e40d38424afe88c8a81cf0e3857d8af4ef077f/sound/ambience/shipambience.ogg). Licensed under CC-BY-SA.
```
to
```yaml
- files: ["shipambience.ogg"]
  license: "CC-BY-SA-4.0"
  copyright: "Taken or edited from source"
  source: "https://github.com/tgstation/tgstation"
```

There are three problems here:
- "Taken or edited from source" is useless and provides no information, the field should be used to provide information about the copyright holder. Generally this just means crediting the authors or describing where the file came from in the event the original author/copyright-holder is unknown. I don't know if the original authors can be tracked in this case, but this is where you should at least mention tg & eris. Similarly for a youtube source it should credit the artist. In the event the artist isn't specified but a valid licence is given, then at least credit the channel/uploader, though in a way that makes it clear they might not have created it (e.g, "Uploaded to youtube by ..."). If the file was modified, just append that to this field as well
- The licence changed? Previously it didn't specify a version (which is already bad), but unless you actually have some way of knowing its 4.0, you can't just assume some number. Though for several files the version was EXPLICITLY set to 3.0 and was just changed to 4.0 (e.g., starlight,ogg, goat_bah.ogg, and many more). So all the licences have to be rechecked to make sure they weren't changed. Ideally, its worth also checking the actual sources to see if the licence is correct. If ship ambience doesn't actually have a proper licence, we can't use it.
- The source field should link directly link to some location where the file can be downloaded or listened to, not the TG github as a whole, that'd be like linking to youtube.com. Generally this is either a perma link to version of the file or the commit that last modified it. The source field should be:

> A valid URL pointing to a location where the file can be downloaded. If you are the creator of the work and don't have an alternate download location (e.g. bandcamp), provide the link to the pull request that added the file to the game. If this is a derivative work, this should be mentioned in the copyright field. If the file has only been lightly modified, just link to the original file. If the file has been heavily modified, link the modified version but provide links to any original files in the copyright field.

**Comment by ficcialfaint** [comment]
Hi ElectroJr, I've been also working on closing that issue (on converting `license.txt`s into RGA), but I was too slow + lazy, so I haven't made a PR. If you're saying

> You can, and should, just open multiple smaller PRs.

then I can probably open PRs with stuff I've converted so far. I have checked links' licenses so they match the actual licenses. I've also marked some files that need to be changed due to either unavailable video (if the source is a YT video) or an incorrect license. So let me know if it's good to also open a PR, I can also try to replace audios that needs to be replaced.
Also it seems like HighTechPuddle missed some files in RGAs, e.g.: `Announcements/attributions.yml`, if you compare the number of audio files in the folder and number of files in the RGA, you'll notice that some are missing (it's cause HighTechPuddle hasn't ensured that original `license.txt`s contain all the files for a folder, which is untrue for some cases), meanwhile the RGAs I've done so far don't have that "issue".

**Comment by HighTechPuddle** [comment]
I see I did a really bad job, I am sorry I rushed it so much I will rework it entirely following the specified rules and will break it down to smaller PRs. Again so sorry for doing such a shit job and taking your time.

**Comment by ElectroJr** [comment]
No need to apologies, your effort is still very much appreciated. And as @ficcialfaint just said they were also working on converting some of the files, which is a another reason why smaller PRs is better, you should probably coordinate so you don't work on the same files at the same time (or just open a draft PR which makes it clear which files you're each working on).

And for faint's comment, yeah some files are just missing attributions and either need to be tracked down or need replacing.

**Pull request closed without merging**